### DISCUSSION TITLE
Is it possible to swap tabs?

### DISCUSSION THREAD

**Discussion by neelchakrabrty** [kovidgoyal/kitty#7826] [Q&A] [answered]
Suppose I have 3 tabs open in kitty, and I want to swap the positions of tab 1 and 3. Is it possible?

**Comment by kovidgoyal** [comment, marked as answer]
Sure move_tab 2 or if you want an general solution write a kitten that
uses the remote control api to count the number of tabs and move_tab
that many times.
